pre-receive-hooks/restrict-master-to-gui-merges.sh

```diff
@@ -5,14 +5,14 @@
 DEFAULT_BRANCH=$(git symbolic-ref HEAD)
 while read -r oldrev newrev refname; do
   if [[ "${refname}" != "${DEFAULT_BRANCH:=refs/heads/master}" ]]; then
-    exit 0
+    continue
   else
     if [[ "${GITHUB_VIA}" != 'pull request merge button' && \
           "${GITHUB_VIA}" != 'pull request merge api' ]]; then
       echo "Changes to the default branch must be made by Pull Request. Direct pushes, edits, or merges are not allowed."
       exit 1
     else
-      exit 0
+      continue
     fi
   fi
 done
```

Comment:
@iamdanfox We're sorry for the delay in merging this and appreciate your contribution.

What do you think about putting in an explicit `exit 0` at the end? It should implicitly return a 0 anyway, but since the hook process depends on `0` code putting it at the would make it all crystal clear.